os/net/mac/tsch/tsch.c

```diff
@@ -686,7 +686,8 @@ tsch_associate(const struct input_packet *input_eb, rtimer_clock_t timestamp)
         tsch_schedule_add_link(sf,
             ies.ie_tsch_slotframe_and_link.links[i].link_options,
             LINK_TYPE_ADVERTISING, &tsch_broadcast_address,
-            ies.ie_tsch_slotframe_and_link.links[i].timeslot, ies.ie_tsch_slotframe_and_link.links[i].channel_offset);
+            ies.ie_tsch_slotframe_and_link.links[i].timeslot,
+            ies.ie_tsch_slotframe_and_link.links[i].channel_offset, 0);
```

Comment:
Is this change required? (If not, I think it should be reverted. But I can live with it if link-based requires it).

Reply:
No, definitely not required. Was not sure what's the intention here, but if removing conflicting links is what we need, I'll revert